src/index.ts

```diff
@@ -179,15 +180,18 @@ export default class MondayConnector extends BaseHttpConnector<
   MondayConnectorEventOptions
 > {
   _sdk: MondaySdk
-  private webhookURL: string
-  private webhookId?: string
+  private webhookURL?: string
   private webhookLastChangedAt?: number
 
   constructor(app: Reshuffle, options: MondayConnectorConfigOptions, id?: string) {
```

Comment:
I'd recommend to register the webhook only if we have events which can be determined in the onStart().
Something like:

```
constructor(app: Reshuffle, options: MondayConnectorConfigOptions, id?: string) {
  super(app, options, id)
  this._sdk = mondaySdk({ token: options.token })
}

async onStart(): Promise<void> {
    const events = Object.values(this.eventConfigurations)
    if (events.length) {
      const base = validateBaseURL(this.configOptions.baseURL)
      const path = validatePath(this.configOptions.webhookPath) || WEBHOOK_PATH

      this.webhookURL = base + path
      this.app.registerHTTPDelegate(path, this)
  }
}
```

Reply:
Fixed